lib/dialects/oracle/query.js

```diff
@@ -0,0 +1,142 @@
+'use strict';
+
+var Utils = require('../../utils')
+  , AbstractQuery = require('../abstract/query')
+  , _ = require('lodash')
+  , autocommit = true;
```

Comment:
This again do not work for oracle driver. Oracle driver not yet support different autocommit options for different sessions.
I have failed feature request about that on driver.
https://github.com/oracle/node-oracledb/issues/159